src/modules/app/route-configs/SprungbrettRouteConfig.js

```diff
@@ -86,7 +86,10 @@ class SprungbrettRouteConfig implements RouteConfig<SprungbrettRouteParamsType,
   getFeedbackTargetInformation = ({ payloads }) => {
     const extras = payloads.extras.data
     const extra = extras && extras.find(extra => extra.alias === SPRUNGBRETT_EXTRA)
-    return ({ alias: SPRUNGBRETT_EXTRA, title: extra && extra.title })
+    if (extra) {
+      return ({ alias: SPRUNGBRETT_EXTRA, title: extra.title })
+    }
+    return ({ alias: SPRUNGBRETT_EXTRA })
```

Comment:
I think this is not correct, is it? I think we should return null instead since the extra is obviously not activated for the current city. Returning null means there will be no feedback option for it.

Reply:
Yeah, you're right. I just "transpiled" the previous code to flow-compliant code :D machine-mode ignored semantics :see_no_evil: